api-schema/xdb.yaml

```diff
@@ -32,7 +32,25 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/ApiErrorResponse'  
+                $ref: '#/components/schemas/ApiErrorResponse'
+  /api/v1/xdb/service/process-execution/stop:
+    post:
+      summary: stop a process execution
+      requestBody:
+        content:
+          'application/json':
+            schema:
+              $ref: '#/components/schemas/ProcessExecutionStopRequest'
+      responses:
+        '200':
+          description: successful operation
+          content: {}
+        '400':
```

Comment:
we also need to return 404 when the process is not existing or not running. (this is the same experience in iWF/Cadence/Temporal). Though in iWF, it's using subStatus in errorResponse. I want to improve it to use HTTP status to be more clear